projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -348,6 +350,7 @@ const IssueScreenWithPath = React.memo(
         initialFrontKey: string | null
     }) => {
         const response = useIssueResponse(path)
+        weatherHider(apolloClient)
```

Comment:
I have moved the check to the issue-screen to ensure that the default edition has had a chance to be set before making a decision on whether or not to hide the weather.

Reply:
Understood. It feels out of place here. One alternative might be to use the event emitter so that when the default edition is set, the calculation then happens.

Or just add a comment here for the reason its here.

Reply:
Just updated PR to instead use an event emitter once the edition cache has been set